docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to form 
+arugments about the proving 
+scheme. 
+
+These arguments are used to 
+check samples of elements, 
+against the original ones. 
+As is noted in other parts
+of the docs, there is the 
+need for a copy check in 
+in SNARK schemes. This is 
+crucial in our zk-SNARK
+scheme, to ensure the 
+correctness of wires and 
+allow us to express circuits
+with less differing vectors 
+per polynomial. 
+
+PLONK uses an argument to check 
+the correctness of a shuffle in 
+its proving algorithm. This allows 
+for the checking of individual 
+wires rather than entire sets. 
+In principle, PLONK can compare sets 
+of polynomials for equality by 
+comparing an `n` degree polynomial 
+with the same `n` degree polynomial 
+where some elements have been 
+rotated. 
+
+Given a set of wire values, at a 
```

Comment:
I think its better to document how the permutation works in general without the proving system, then at the end, we show how it is used in PLONK by introducing terminology such as "wire values"